src/autocq.c

```diff
@@ -77,8 +84,15 @@ int auto_cq(void)
 		    stoptx();
 		    break;
 		}
+		gettimeofday(&timecnt, NULL);
```

Comment:
While correct, I think that is too complex. Just calling time_update() here does the whole job (same in line 105). It has the additional value that other time related work gets also done during auto_cq.